labgrid/driver/dediprogflashdriver.py

```diff
@@ -0,0 +1,72 @@
+import os.path
+import logging
+import attr
+
+from ..resource import NetworkDediprogFlasher
+from ..factory import target_factory
+from ..step import step
+from .common import Driver, check_file
+from ..util.managedfile import ManagedFile
+from ..util.helper import processwrapper
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class DediprogFlashDriver(Driver):
+    """ The DediprogFlashDriver uses the dediprog utility to write an image
+    to a raw rom. The driver is a pure wrapper of the dpcmd utility"""
+    bindings = {
+        'dediprog_resource': {"DediprogFlasher", NetworkDediprogFlasher},
+    }
+
+    image = attr.ib(default=None)
```

Comment:
```suggestion
    image = attr.ib(validator=attr.validators.instance_of(str), default=None)
```
The docs say that we expect a string here, so let's check for that.